provider/runbook_resource.go

```diff
@@ -170,10 +189,20 @@ func createResourceFireHydrantRunbook(ctx context.Context, d *schema.ResourceDat
 	// Get the API client
 	firehydrantAPIClient := m.(firehydrant.Client)
 
+	attachmentRuleMap := map[string]interface{}{}
+	attachmentRule := d.Get("attachment_rule").(string)
+	if attachmentRule != "" {
+		err := json.Unmarshal([]byte(attachmentRule), &attachmentRuleMap)
+		if err != nil {
+			return diag.Errorf("Error converting attachment_rule %s to map: %v", attachmentRule, err)
+		}
+	}
+
```

Comment:
bit of a nit: could you bring this down to below line 208 to keep all the processing of attributes together?